frontend/appflowy_flutter/lib/user/presentation/screens/sign_in_screen/widgets/third_party_sign_in_buttons.dart

```diff
@@ -30,89 +30,108 @@ class ThirdPartySignInButtons extends StatelessWidget {
 
     if (isMobile) {
       // ThirdPartySignInButtons in mobile
-      return Column(
-        children: [
-          _ThirdPartySignInButton(
-            isMobile: true,
-            icon: FlowySvgs.google_mark_xl,
-            labelText: LocaleKeys.signIn_LogInWithGoogle.tr(),
-            onPressed: () {
-              _signInWithGoogle(context);
-            },
-          ),
-          const SizedBox(height: 8),
-          _ThirdPartySignInButton(
-            isMobile: true,
-            icon: isDarkMode
-                ? FlowySvgs.github_mark_white_xl
-                : FlowySvgs.github_mark_black_xl,
-            labelText: LocaleKeys.signIn_LogInWithGithub.tr(),
-            onPressed: () {
-              _signInWithGithub(context);
-            },
-          ),
-          const SizedBox(height: 8),
-          _ThirdPartySignInButton(
-            isMobile: true,
-            icon: isDarkMode
-                ? FlowySvgs.discord_mark_white_xl
-                : FlowySvgs.discord_mark_blurple_xl,
-            labelText: LocaleKeys.signIn_LogInWithDiscord.tr(),
-            onPressed: () {
-              _signInWithDiscord(context);
-            },
-          ),
-        ],
+      return LayoutBuilder(
+        builder: (context, constraints) => Column(
+          children: [
+            _ThirdPartySignInButton(
+              buttonWidth: constraints.maxWidth,
+              isMobile: true,
+              icon: FlowySvgs.google_mark_xl,
+              labelText: LocaleKeys.signIn_LogInWithGoogle.tr(),
+              onPressed: () {
+                _signInWithGoogle(context);
+              },
+            ),
+            const SizedBox(height: 8),
```

Comment:
We have these `VSpace` and `HSpace` (not by my invention 😂)
```suggestion
            const VSpace(8),
```